dev/all_tests.yaml

```diff
@@ -333,3 +333,11 @@ tests:
     api: gate
     args:
       alias: [standard_dcos_provider_params]
+
+  plugin_stage_test:
+    requires:
+      services: [gate, orca]
+    api: gate
+    args:
+      stage_name: randomWait
+      stage_params: '{ "maxWaitTime": 5 }'
```

Comment:
This line is the only difference between this PR and the previous PR. In the previous PR, 
```
stage_params: |
  {
     "maxWaitTime: 5
  } 
```
and was parsed as `{\n "maxWaitTime": 5 \n}`. 

`json.loads(...)` parses that correctly, but I recreated the error when passing in the flag on the command line. I think the shell hits the newline and only passes `{` as a param into my test. I think the test runner shells out to start each test... so this adds up.